.yaydoc.yml

```diff
@@ -3,26 +3,19 @@ metadata:
   projectname: "Susi Server"
   version: development
 build:
-  theme: sphinx_fossasia_theme   
-  source: docs
+  theme:
+    name: sphinx_fossasia_theme
   logo: images/susi-logo-original.svg
   subproject:
     - url: https://github.com/fossasia/susi_android
-      source: docs
     - url: https://github.com/fossasia/chat.susi.ai
-      source: docs
     - url: https://github.com/fossasia/susi_tweetbot
-      source: docs
     - url: https://github.com/fossasia/susi_fbbot
-      source: docs
     - url: https://github.com/fossasia/susi_slackbot
-      source: docs
     - url: https://github.com/fossasia/susi_desktop
       source: .
     - url: https://github.com/fossasia/susi_iOS
-      source: docs
     - url: https://github.com/fossasia/susi_telegrambot
-      source: docs
 publish:
   ghpages:
     docurl: dev.susi.ai
```

Comment:
I have decreased the space between the side bar. I have decreased the font size also.
preview: http://yaydoc.herokuapp.com/preview/rbalajis25@gmail.com/08f7749e-1db6-4f1e-8426-5970deeb0c55_preview/